app/src/main/database/migrations/20250821184819_add_item_fetch.sql

```diff
@@ -0,0 +1,23 @@
+-- migrate:up
+ALTER TABLE items
+ADD COLUMN progress INTEGER;
+
+ALTER TABLE items
+ADD COLUMN status TEXT NOT NULL DEFAULT 'INITIAL' CHECK(status IN ('INITIAL', 'IN_PROGRESS', 'COMPLETE', 'FAILED', 'PAUSED'));
```

Comment:
You might've already considered this, but using an integer for status would be a bit more efficient and also not require schema changes for further statuses. But using text is far nicer for debugging and it's not like the space savings would actually help us, so I don't actually have a preference.